index.html

```diff
@@ -1,11 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Product Landing Page</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="landingPage.css">
 </head>
 <body>
-  
+  <!-- Navigation bar -->
+    <header>
+        <nav>
+            <a href="#Home"><img src="logo.png" alt="Home"></a>
+            <ul>
+                <li><a href="#Home">Home</a></li>
+                <li><a href="#Shop">Shop</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <div class="hero">
+      <img id="hero-hats" src="heroHats.jpeg" alt="Hero Hats">
+      <div class="hero-text">
+        <h1>Tip Top Hats</h1>
+        <p>All handcrafted in Australia</p>
+        <button>Shop Now</button>
+      </div>
+    </div>
+
+    <!-- Description of website -->
+    <section class="description">
+      <p>Tip Top Hats was founded in 1534 by an Australian farmer, Henry Tiptop. The original hat was crafted from kangaroo hide. The techniques of our hat crafting has since been perfected and past down for many generations. We have even branched out into a new line of carbon-free and vegan-friendly hats to accommodate all lifestyles.</p>
+    </section>
+
+    <!-- Footer section -->
+    <footer>
+        <section class="faq">
+            <a href="#faq"><h2>FAQ</h2></a>
+        </section>
+        <section class="contact">
+            <a href="#contact"><h2>Contact Us</h2></a>
+        </section>
+        <section class="about-us">
+            <a href="#about"><h2>About Us</h2></a>
+        </section>
```

Comment:
**[Question]**
Are we going to have descriptions after each of these headings later on?